### PR TITLE
Re-format ui command

### PR DESCRIPTION
**Pull request by isirk**

### CODE REVIEW

File: cogs/utility.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ async def user_info(self, ctx, *, user: discord.User = None):
         if not db_user:
             raise NoSomething(user)
 
-        embed = discord.Embed(colour=await self.embed_colour(ctx))
+        embed = discord.Embed(colour=user.color) # Make it so that it shows the color of the role as the embed color, though its mostly gonna be pink
```
Comment (Cairo2k18) [inline cogs/utility.py:72]: What actually changes here?

File: cogs/utility.py [hunk 2]
```diff
@@ -104,7 +104,8 @@ async def user_info(self, ctx, *, user: discord.User = None):
 
         embed.add_field(name=f"{self.bot.settings['emoji']['shadows']} Username", value=f"{db_user['fullUsername']}"
                                                                                         f" {' '.join(badges)}")
-        embed.add_field(name=f"{self.bot.settings['emoji']['id']} ID", value=db_user["_id"])
+        # embed.add_field(name=f"{self.bot.settings['emoji']['id']} ID", value=db_user["_id"]) # You can change this idc
+        embed.set_footer(text=f"ID: db_user["_id"]")
```
Comment (Cairo2k18) [inline cogs/utility.py:108]: nothing really wrong with this, but I will probably get others thoughts first

**Comment by ImVaskel** [comment]
Do the users discord status instead: statuses = {'online': discord.Color.green(), 'dnd': discord.Color.red(), 'idle': discord.Color.gold(),
                    'offline': discord.Color.dark_grey()}

File: cogs/utility.py (hunk 2 above)
Comment (isirk) [reply cogs/utility.py:108]: it basically reformats what was already there

**Comment by isirk** [comment]
that would be cool

File: cogs/utility.py (hunk 2 above)
Comment (TheMoksej) [reply cogs/utility.py:108]: That's what you call invalid syntax and incorrect f-string. It'd actually be f"ID: {db_user['_id']}" _ik this is closed but still letting you know cause why not._

**Pull request closed without merging**